plenum/server/node.py

```diff
@@ -756,19 +771,16 @@ def send_ledger_status_to_newly_connected_node(self, node_name):
 
     def newNodeJoined(self, txn):
         self.setF()
+        new_replicas = self.adjustReplicas()
         if self.adjustReplicas() > 0:
-            self.decidePrimaries()
-        # TODO: Should tell the client after the new node has synced up its
```

Comment:
Because it was based on the assumption that let the node catchup until then can stop sending requests to it, but the nodes can stash client's request anyway. Also we are sending this info to client in `onPoolMembershipChange`